### PR TITLE
EDCD.EliteDangerousMarketConnector version 5.2.4

### PR DESCRIPTION
**Pull request by Rolling2405**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/43687)

### CODE REVIEW

**Comment by Rolling2405** [comment]
Seems to be stuck scanning

**Comment by OfficialEsco** [comment]
Close and reopen the PR to rerun the pipelines (same button)

File: manifests/e/EDCD/EliteDangerousMarketConnector/5.2.4/EDCD.EliteDangerousMarketConnector.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: EDCD.EliteDangerousMarketConnector
+PackageVersion: 5.2.4
```
Comment (ItzLevvie) [inline manifests/e/EDCD/EliteDangerousMarketConnector/5.2.4/EDCD.EliteDangerousMarketConnector.installer.yaml:5]:
```suggestion
PackageVersion: 5.2.4.0
```

File: manifests/e/EDCD/EliteDangerousMarketConnector/5.2.4/EDCD.EliteDangerousMarketConnector.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: EDCD.EliteDangerousMarketConnector
+PackageVersion: 5.2.4
```
Comment (ItzLevvie) [inline manifests/e/EDCD/EliteDangerousMarketConnector/5.2.4/EDCD.EliteDangerousMarketConnector.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 5.2.4.0
```

File: manifests/e/EDCD/EliteDangerousMarketConnector/5.2.4/EDCD.EliteDangerousMarketConnector.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: EDCD.EliteDangerousMarketConnector
+PackageVersion: 5.2.4
```
Comment (ItzLevvie) [inline manifests/e/EDCD/EliteDangerousMarketConnector/5.2.4/EDCD.EliteDangerousMarketConnector.yaml:5]:
```suggestion
PackageVersion: 5.2.4.0
```

File: manifests/e/EDCD/EliteDangerousMarketConnector/5.2.4/EDCD.EliteDangerousMarketConnector.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: EDCD.EliteDangerousMarketConnector
+PackageVersion: 5.2.4
+PackageLocale: en-US
+Publisher: EDCD - Elite Dangerous Community Developers
```
Comment (ItzLevvie) [inline manifests/e/EDCD/EliteDangerousMarketConnector/5.2.4/EDCD.EliteDangerousMarketConnector.locale.en-US.yaml:7]:
```suggestion
Publisher: EDCD
```

File: manifests/e/EDCD/EliteDangerousMarketConnector/5.2.4/EDCD.EliteDangerousMarketConnector.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: EDCD.EliteDangerousMarketConnector
+PackageVersion: 5.2.4
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/EDCD/EDMarketConnector/releases/download/Release%2F5.2.4/EDMarketConnector_win_5.2.4.msi
+  InstallerSha256: 5DDD2629EF1AD70D735CE36AF1525C0765A72BBBC3411DA8647ED09ADF20BC33
+  ProductCode: '{DB995DF6-4A0A-4ED9-BC2F-0A82B94EBD7B}'
```
Comment (ItzLevvie) [inline manifests/e/EDCD/EliteDangerousMarketConnector/5.2.4/EDCD.EliteDangerousMarketConnector.installer.yaml:12]:
```suggestion
  ProductCode: '{DB995DF6-4A0A-4ED9-BC2F-0A82B94EBD7B}'
  Scope: machine
```

File: manifests/e/EDCD/EliteDangerousMarketConnector/5.2.4/EDCD.EliteDangerousMarketConnector.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: EDCD.EliteDangerousMarketConnector
+PackageVersion: 5.2.4
+PackageLocale: en-US
+Publisher: EDCD - Elite Dangerous Community Developers
+PackageName: Elite Dangerous Market Connector
+License: GPL-2.0 License
+ShortDescription: Mod/ Plugin for Elite Dangerous that helps contribute data to EDDB.IO
```
Comment (ItzLevvie) [inline manifests/e/EDCD/EliteDangerousMarketConnector/5.2.4/EDCD.EliteDangerousMarketConnector.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Mod/Plugin for Elite Dangerous that helps contribute data to EDDB.IO
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**